backups/init.sls

```diff
@@ -1,5 +1,12 @@
 {% from 'backups/map.jinja' import backup_root, mail_on_success, mail_to, mail_from with context %}
 
+{$backup_root}:
```

Comment:
I think you need these Jinja markers:
```suggestion
{{ backup_root }}:
```

You'll also want to create subdirectories for each of the database identifiers. The best place is probably to do that in this loop: https://github.com/forumone/f1-backups-formula/blob/fd9da84a46fe663f09b52a99daf69d06e2fd4ae9/backups/init.sls#L34

Reply:
update pushed and ready for review. 
the makedirs true for the identifier dir makes the first file.directory i added unneeded, right?